shared/menubar/remote-proxy.desktop.tsx

```diff
@@ -9,13 +9,14 @@ import * as Electron from 'electron'
 import {intersect} from '../util/set'
 import useSerializeProps from '../desktop/remote/use-serialize-props.desktop'
 import {serialize} from './remote-serializer.desktop'
-import {isDarwin, isWindows} from '../constants/platform'
+import {isDarwin, isWindows, isLinux} from '../constants/platform'
 import {isSystemDarkMode} from '../styles/dark-mode'
 import {uploadsToUploadCountdownHOCProps} from '../fs/footer/upload-container'
 import {ProxyProps, RemoteTlfUpdates} from './remote-serializer.desktop'
 import {mapFilterByKey} from '../util/map'
 import {memoize} from '../util/memoize'
 import shallowEqual from 'shallowequal'
+import {useInterval} from '../common-adapters/use-timers'
```

Comment:
?

Reply:
Ah, I used this to help with figuring out. Forgot to remove it. Will change